app/src/main/dist/config/crate.yml

```diff
@@ -102,9 +102,9 @@ auth:
 
 # When trust based authentication is used, the server just takes the username
 # provided by the client as is without further validation. The HTTP
-# implementation takes the value of the `X-USER` request header as the
-# username. Since a user is always required for trust based authentication, a
-# default user (in case `X-USER` is not set) can be defined as follows:
+# implementation extracts the username from the standard HTTP Basic Auth Header
+#`Authorization` request header. Since a user is always required for trust based
+# authentication, a default user (in case `Authorization` is not set) can be defined as follows:
```

Comment:
In case the `Authorize` header is not set, a default username can be specified as follows:

Reply:
@haudi but the  header is `Authorization`... or?